src/javascripts/vendor/iframeResizer.js

```diff
@@ -0,0 +1,9 @@
+/*! iFrame Resizer (iframeSizer.min.js ) - v3.5.15 - 2017-10-15
```

Comment:
Is this file meant to still be committed?

Reply:
Nope, I shall remove it.